prod-deploy/kv-mall-manifest/kv-mall.yaml

```
@@ -20,28 +19,28 @@ spec:
         app: analytics
     spec:
       containers:
-        - name: analytics
-          image: registry.odigos.io/kv-mall-analytics:v0.6
-          env:
-            - name: KAFKA_ADDRESS
-              value: kafka-service.kv-mall-infra:9092
-            - name: MEMCACHED_ADDR
-              value: memcached.kv-mall-infra:11211
-            - name: CASSANDRA_ADDR
-              value: cassandra.kv-mall-infra:9042
-            - name: POSTGRESQL_ADDR
-              value: postgresql://postgres:postgres@postgres.kv-mall-infra:5432/postgres?sslmode=disable
-          startupProbe:
-            httpGet:
-              path: /health
-              port: 8081
-            initialDelaySeconds: 5
-            periodSeconds: 5
-            failureThreshold: 100
-          ports:
-            - name: http
-              containerPort: 8081
-              protocol: TCP
+      - name: analytics
+        image: dev/analytics:dev
```

Note the dev tag on production deployment:
```suggestion
        image: registry.odigos.io/kv-mall-analytics:v0.xx
```